docs/docs/feature-library/datadog-log-integration.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_label: datadog-log-integration
+title: datadog-log-integration
+---
+
+This feature forwards logs emitted by the Flex Project Template to Datadog.
+
+### Configuration
+
+This feature allows the following configuration settings.
+
+![datadog-log-integration settings](../../static/img/features/datadog-log-integration/settings.png)
+
+| Setting | Description |
+| --------| ------------|
+| Api key | Your Datadog Account [client token](https://docs.datadoghq.com/account_management/api-app-keys/#client-tokens)
+| Log Level | The minimum log level to send to Datadog. See [Logging](/flex-project-template/building/template-utilities/logging) for more details.
+| Intake Region | The [Datadog Site](https://docs.datadoghq.com/getting_started/site/) for your account.
```

Comment:
Maybe we could list the valid values that we support for this setting.

Reply:
Done!